app/models/charts/comparison_chart_service.rb

```diff
@@ -0,0 +1,85 @@
+class ComparisonChartService
```

Comment:
Still a pretty meaty class.  But I like that is has really only when responsibility (I think):  Create the charts from the SelfReflection data analysis class and pass them to the view.  Thoughts? @jmejia